### PR TITLE
Add el 8 builders

### PR DESCRIPTION
**Pull request by vinay-satish**
### Description

Adding el 8 builder and tester to the adhoc pipeline.

### Issues Resolved

Testing the perl upgrade - https://github.com/chef/chef-server/pull/3261

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

**Review by lbakerchef — COMMENTED** [review]
Is there a build or test to show that this is working?

**Comment by lbakerchef** [comment]
> Yes, forgot to add them. Thank you! Adhoc build - https://buildkite.com/chef/chef-chef-server-main-omnibus-adhoc/builds/4225

No worries.

I noticed a run time on the build of 8.5 hours!  I wonder what is causing that?

**Comment by lbakerchef** [comment]
Re-ran the build and it passed in 2 hours:
https://buildkite.com/chef/chef-chef-server-main-omnibus-adhoc/builds/4230

**Comment by jashaik** [comment]
https://buildkite.com/chef/chef-umbrella-main-chef-server/builds/1194

File: omnibus_overrides.rb [hunk 1]
```diff
@@ -7,12 +7,13 @@
...
 override :ruby, version: "2.7.5"
-override :perl, version: "5.18.1"
+override :perl, version: "5.34.0"
 override :redis, version: "5.0.14"
 override :runit, version: "2.1.1" #standalone upgrade is failing, Needs to be reverted to 2.1.2 after fixing the umbrella
 override :sqitch, version: "0.973"
 
-override :cpanminus, version: "1.7004" # 1.9019 breaks installs currently
+#removing the override for perl update
+#override :cpanminus, version: "1.7004" # 1.9019 breaks installs currently
```
Comment (jashaik) [inline omnibus_overrides.rb:16]: remove these comments

Comment (vinay-satish) [reply omnibus_overrides.rb]: update them. Thanks for the review!

**Pull request merged**